src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,20 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine truck = new Truck();
 
+        Machine[] machines = new Machine[] {bulldozer, excavator, truck};
+
+        for (Machine machine : machines) {
+            workAndStop(machine);
+        }
+    }
+
+    public static void workAndStop(Machine machine) {
```

Comment:
it is redundant to create a separate method for starting and stopping machines.
let's call `machine.doWork()` and `machine.stopWork()` at the `for` loop